### PR TITLE
Setup CD

### PR DESCRIPTION
**Pull request by Bachmann1234**
Will automatically deploy on tagging. Has the benefits of being less work, less error prone.

Now I wont forget to tag a release :-P

### CODE REVIEW

**Comment by Bachmann1234** [comment]
Lot of the changes were done by travis's cli tool which handled the encryption of the pypi info

File: .travis.yml [hunk 1]
```diff
@@ -8,18 +6,20 @@ matrix:
...
-
+- flake8 .
 script: pytest
-
 after_success: coveralls
+deploy:
+  provider: pypi
+  user: "matt.bachmann"
+  password:
+    secure: i68kRacnIViEF+4t8/r8L+w8RJv/WeGIIf9l+s6lSXxkmedQPVogCWLyFZXLCHZP4a9vp/ho/UzP9phB9Gv9B/KBJ/cXqgGrTzamyBNvFu4MUXjUWSfQcnPLE7Wyq+DYzoFbGTnk1F3kQKhxUkZ2RZu3v9NXZN78lSvhexyTv+QXk9wIZTP2gYANt+a/grhVHqZ0YQLukv3iOO3yKLtvh0Td2MQJK7y7J4gC11Z9qv0AGosIWVnVUocxVCny5H6zXtmwTefwbj9ZoeeDCsf/sPrWen7plHJeClmDRejjwByourzbv2NMV6IL3BIP62Bl/uJqQQxaLGMce2vdduA88OuYfoom3yJyEr1aW6yjbTK13Zrk5yQu6AUdZtYcqD05DZekwaNpozr4ua1iYL7naYhfQk8qM3K6cxbHih2Ri7nM+dyECB24VQhuIbILqmEvJhiQmlzLwRognCj4sd9u9EwxAPJlTxwiZXMbtCu8nWazL7FVPO/Imc63kbPZy4EPgD8MsV7lPJbRNlkINT6jpd6o0pC9iVq6fKio0YlxUxQ+VKTahtVLlLIxI1emUZRjrSiWHh81V3prbIugICF8JgRbuK0Kw8xxJfMxwtM/UbBz6jfdrv0QD07WlIf71v6/ZuGakVPzKs9yJU4m5g52MHuFOEMH5jD7jS00Rg24lSA=
```
Comment (Jamim) [inline .travis.yml:23]: @Bachmann1234 Is it really ok to share this?

Comment (Jamim) [reply .travis.yml:23]: Ah, I believe it's encrypted already, so never mind :smile:

Comment (Bachmann1234) [reply .travis.yml:23]: Its encrypted on their side. So im trusting travis with it but its not shared publicly. https://docs.travis-ci.com/user/encryption-keys/ 

https://docs.travis-ci.com/user/deployment/pypi/

**Pull request merged**